charts/vector/README.md

```diff
@@ -140,7 +140,7 @@ helm install --name <RELEASE_NAME> \
 | persistence.size | string | `"10Gi"` | Specifies the size of PersistentVolumeClaims |
 | podAnnotations | object | `{}` | Set annotations on Vector Pods |
 | podDisruptionBudget.enabled | bool | `false` | Enable a PodDisruptionBudget for Vector |
-| podDisruptionBudget.maxUnavailable | int | `1` | The number of Pods that can be unavailable after an eviction |
+| podDisruptionBudget.maxUnavailable | string | `nil` | The number of Pods that can be unavailable after an eviction |
```

Comment:
This change seems to be the same as it is in https://github.com/vectordotdev/helm-charts/pull/104?

Reply:
I had merged in develop to fix CI